jupyter-lab/tests/test_api.py

```diff
@@ -0,0 +1,194 @@
+import os
+
+# import random as rnd
+import time
+import datetime
+import uuid
+import json
+import requests
+from requests.exceptions import ConnectionError
+from websocket import create_connection
+import pytest
+import docker
+
+
+# Settings
+PORT = 8888  # the jupyter-lab port
+HOST = "localhost:8888"  # the host
+NOTEBOOK_PATH = "tests/basic.ipynb"  # the relative path to the jupyter test notebook
+TOKEN = "a268cff61a101aaefe53b02b5a787ddfc0e07d4119154bff"  # the token to use
+TIMEOUT_CALL = 5  # the timeout in seconds of the client request call
+
+
+client = docker.from_env()
+
+container = client.containers.run(
+    os.environ["IMAGE_NAME"],
+    command=f"start-notebook.sh --NotebookApp.token='{TOKEN}'",
+    ports={
+        f"{PORT}/tcp": PORT,
+    },
+    detach=True,
+)
+
+time.sleep(20)
+
+container.reload()
+
+
+# Tests
+def test_jupyterlab_status_is_running():
+    """Test that the jupyter-lab container is running."""
```

Comment:
Nice that you comment the code like this. 
I think we are using different conventions for commenting. I usually use `# comment`.
We should decide on a convention and stick to that. I'll create an issue regarding this.

Reply:
I use # for comments but docstrings reStructuredText for documenting classes, methods etc.

Reply:
Agree. We can discuss hot to comment and document the same way.